bin/wazo-dist-upgrade-bullseye-check

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python3
```

Comment:
Do these new files need to be added to `wazo-dist-upgrade.install`? They currently don't seem to be added to the package.

Reply:
Yes, fixed